Api/Controllers/TestController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Reservant.Api.Identity;
+using Reservant.Api.Models;
+
+namespace Reservant.Api.Controllers;
+
+[ApiController, Route("/test")]
+public class TestController(UserManager<User> userManager): Controller
+{
+    [HttpGet("/restaurant-owner-only")]
```

Comment:
W ścieżkach końcówek nie trzeba pisać / na początku, bo wtedy to nadpisuje ścieżkę kontrolera i jest po prostu np. '/customer-only' zamiast '/test/customer-only'